### ISSUE TITLE
The wallpaper set on the tablet is very blurry.

### ISSUE THREAD

**Issue by bigsaltyfishes** [StatiXOS/android_packages_apps_WallpaperPicker2#1]
I built a GSI for my tablet, and when I open Wallpaper & style in settings or from the launcher, the ThemePicker will be displayed in a split screen with the Settings app, and it seems that the WallpaperPicer component will incorrectly calculate the screen resolution causing the wallpaper to be cropped incorrectly, so the wallpaper looks blurry. And it won't appear in this situation after the wallpaper is set up by the ThemePicker in a non-split screen state (Open it from AOSP WallpaperPicker).

Set Wallpaper in Settings
![20220324112349](https://user-images.githubusercontent.com/15278036/159835903-246563f4-44a1-466b-adc1-b91b0084acf7.png)

Set Wallpaper in AOSP WallpaperPicker:
![20220324112354](https://user-images.githubusercontent.com/15278036/159835978-87f9332a-9fd0-4fb8-b546-223b964a8c1e.png)\

Settings on Table:
![20220324112806](https://user-images.githubusercontent.com/15278036/159836287-53a4bff6-dc7b-4235-b552-b30a3c14f052.png)

**Comment by anayw2001** [comment]
Yes, we are aware, we will likely have to ship landscape wallpapers to address this issue.
